BUILD.sh

```diff
@@ -86,7 +86,7 @@ EOD
   esac
 done
 
-set -xv
+# set -xv
```

Comment:
@CameronRutherford, could you please comment on this and resolve, if needed?

Reply:
I commented this out because I was getting annoyed by the excessive output from build scripts. We are at a point in development now where this shouldn't be necessary for every build, but perhaps I am misguided.

An alternative that we decided on in ExaGO was to have a `-v|--verbose` option to pass to the build script that would toggle this sort of thing. https://gitlab.pnnl.gov/exasgd/frameworks/exago/-/blob/develop/buildsystem/build.sh#L185 - thanks again @jaelynlitz for implementing that.